src/index.spec.ts

```diff
@@ -62,4 +63,29 @@ describe('cassette', () => {
       });
     });
   });
-})
\ No newline at end of file
+
+  it('does not recored blacklisted host', async () => {
```

Comment:
```suggestion
  it('does not record blacklisted host', async () => {
```

Reply:
thanks for the quick reply. fixed the typo and also changed Url().host to Url().hostname seeing that .host also holds port name, which I don't want.